woocommerce-gateway-ebanx/gateways/class-wc-ebanx-debit-card-gateway.php

```diff
@@ -132,7 +138,47 @@ protected function save_order_meta_fields($order, $request)
 		parent::save_order_meta_fields($order, $request);
 
 		update_post_meta($order->id, '_cards_brand_name', $request->payment->payment_type_code);
-		update_post_meta($order->id, '_masked_card_number', $_POST['ebanx_masked_card_number']);
```

Comment:
We're not using superglobals anymore, please replace `$_POST[<value>]` with `WC_EBANX_Request::read(<value>)`